Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/RectangularBeamProfile.h

```diff
@@ -30,6 +30,15 @@ class MANTID_ALGORITHMS_DLL RectangularBeamProfile final : public IBeamProfile {
   IBeamProfile::Ray generatePoint(Kernel::PseudoRandomNumberGenerator &rng,
                                   const Geometry::BoundingBox &bounds) const override;
   Geometry::BoundingBox defineActiveRegion(const Geometry::BoundingBox &) const override;
+  /// Returns the min point of the profile
+  inline Kernel::V3D minPoint() const { return Kernel::V3D{m_min[0], m_min[1], m_min[2]}; }
+  /// Returns the max point of the profile
+  inline Kernel::V3D maxPoint() const {
+    auto maxPt = Kernel::V3D{m_min[0], m_min[1], m_min[2]};
```

Comment:
Can this be changed to
```suggestion
    auto maxPt = minPoint();
```
?